sklearn/utils/tests/test_utils.py

```diff
@@ -1,139 +1,7 @@
-import warnings
-
-import numpy as np
 import pytest
 
-from sklearn.utils import (
-    check_random_state,
-    column_or_1d,
-    deprecated,
-    safe_mask,
-    tosequence,
-)
-from sklearn.utils._missing import is_scalar_nan
-from sklearn.utils._testing import assert_array_equal, assert_no_warnings
-from sklearn.utils.fixes import CSR_CONTAINERS
-from sklearn.utils.validation import _is_polars_df
-
-
-def test_make_rng():
-    # Check the check_random_state utility function behavior
-    assert check_random_state(None) is np.random.mtrand._rand
-    assert check_random_state(np.random) is np.random.mtrand._rand
-
-    rng_42 = np.random.RandomState(42)
-    assert check_random_state(42).randint(100) == rng_42.randint(100)
-
-    rng_42 = np.random.RandomState(42)
-    assert check_random_state(rng_42) is rng_42
-
-    rng_42 = np.random.RandomState(42)
-    assert check_random_state(43).randint(100) != rng_42.randint(100)
-
-    with pytest.raises(ValueError):
-        check_random_state("some invalid seed")
-
-
-def test_deprecated():
-    # Test whether the deprecated decorator issues appropriate warnings
-    # Copied almost verbatim from https://docs.python.org/library/warnings.html
-
-    # First a function...
-    with warnings.catch_warnings(record=True) as w:
-        warnings.simplefilter("always")
-
-        @deprecated()
-        def ham():
-            return "spam"
-
-        spam = ham()
-
-        assert spam == "spam"  # function must remain usable
-
-        assert len(w) == 1
-        assert issubclass(w[0].category, FutureWarning)
-        assert "deprecated" in str(w[0].message).lower()
-
-    # ... then a class.
-    with warnings.catch_warnings(record=True) as w:
-        warnings.simplefilter("always")
-
-        @deprecated("don't use this")
-        class Ham:
-            SPAM = 1
-
-        ham = Ham()
-
-        assert hasattr(ham, "SPAM")
-
-        assert len(w) == 1
-        assert issubclass(w[0].category, FutureWarning)
-        assert "deprecated" in str(w[0].message).lower()
```

Comment:
This one I just removed it because it's redundant and strickly less exhaustive than the `test_deprecated` function already in `test_deprecation.py`.